src/cow-react/common/hooks/useCancelOrder/useSendOnChainCancellation.test.ts

```diff
@@ -0,0 +1,120 @@
+import { renderHook } from '@testing-library/react-hooks'
+import { useSendOnChainCancellation } from './useSendOnChainCancellation'
+import { Order } from 'state/orders/actions'
+import { useWalletInfo } from '@cow/modules/wallet'
+import { useRequestOrderCancellation, useSetOrderCancellationHash } from 'state/orders/hooks'
+import { useTransactionAdder } from 'state/enhancedTransactions/hooks'
+import { useEthFlowContract, useGP2SettlementContract } from 'hooks/useContract'
+import { NATIVE_CURRENCY_BUY_TOKEN } from 'constants/index'
+import { BigNumber } from '@ethersproject/bignumber'
+import { COW } from 'constants/tokens'
+
+const chainId = 1
+const settlementCancellationTxHash = '0xcfwj23g4fwe111'
```

Comment:
Lol, it looks like you hit your keyboard hard to randomize this address

Reply:
Ha ha :) Yes, why not?